app/views/projects/get_projects.html.erb

```diff
@@ -0,0 +1,38 @@
+<link href="/css/search.css" rel="stylesheet">
+<div class="container">
+  <h1>Circuitverse Projects:</h1>
+  <h2>Searched Tags: <%= params[:tag]%></h2>
+  <% @projects.each do |project| %>
+    <div class="row feature m-4">
+      <div class="col-md-9">
+        <h3 class="card-title"><%= project.name %></h3>
+        <div class="description">
+          <%= project.description %>
+        </div>
+        <div class="row mt-3">
+          <div class="col-4">
+            <div class="mt-3 mb-3">
+              <a class="btn btn-dark btn-md" target="_blank" href="<%= user_project_path(project.author, project) %>" role="button">View</a>
+            </div>
+          </div>
+          <div class="col-8 d-flex justify-content-end">
+            <div class="row">
+              <div class="col-sm-auto p-1">
+                <%= image_tag project.author.profile_picture.url(:medium) , :style => "height:60px;width:60px;", :class => ' img-responsive thumbnail rounded-circle'%>
+              </div>
+              <div class="col p-3">
+                <small class="text-muted"><%= link_to project.author.name, project.author %></small>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="description">Tags:
+          <%= raw project.tags.map(&:name).map { |t| link_to t, tag_path(t),{:style=>"background-color: #f1f1f1; color: #000000", :class=>"badge"} }.join(', ') %>
+        </div>
+      </div>
+      <div class="col-md-3">
+        <img src="<%= project.image_preview.url %>" alt="<%= project.name %>" class="img-fluid img-thumbnail">
+      </div>
+    </div>
+  <% end %>
+</div>
```

Comment:
Can we add a blank line at the end. No linting reasons, it just looks better. Also no ugly messages such as `No newline at the end`